client/src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Layout, Button } from 'antd';
+import React from 'react';
+import styles from './Sidebar.module.css';
+
+const { Header, Footer } = Layout;
+
+const Sidebar: React.FC = () => {
+  const userName: string = 'superadmin';
+
+  return (
+    <Layout className={`${styles.sidebar} ${styles['sidebar-layout']}`}>
```

Comment:
To apply multiple classes it's better to use this syntax: className={[styles.classOne, styles.classTwo].join(' ')}. And better use camelCase in classes names - you avoid the styles['class-name'] syntax.